.claude/common-patterns.md

```
@@ -297,10 +297,138 @@ func (c *EntityClient) CreateEntity(ctx context.Context, req *CreateEntityReques
     entity := &entities.Entity{
         // Map from request
     }
+```
+
+## Production Authentication Security Patterns
+
+### JWT Token Management
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)